### ISSUE TITLE
Secret length for SignedCookieSessionFactory

### ISSUE THREAD

**Issue by hongyuan1306** [Pylons/pyramid#3234]
The doc of `SignedCookieSessionFactory` in pyramid.session says:

```
    ``secret``
      A string which is used to sign the cookie. The secret should be at
      least as long as the block size of the selected hash algorithm. For
      ``sha512`` this would mean a 128 bit (64 character) secret.  It should
      be unique within the set of secret values provided to Pyramid for
      its various subsystems (see :ref:`admonishment_against_secret_sharing`).
```

Shouldn't it be 512 bit (64 characters)??

**Comment by bertjwregeer** [comment]
You are correct.

**Comment by mmerickel** [comment]
Fixed in #3244.
